etna/transforms/math/binary_operator.py

```diff
@@ -0,0 +1,194 @@
+from enum import Enum
+from typing import List
+from typing import Optional
+
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.transforms.base import ReversibleTransform
+
+
+class BinaryOperator(str, Enum):
+    """Enum for mathematical operators from pandas."""
+
+    add = "+"
```

Comment:
Let's make a documentation for these operations to show which symbol do they use like:
```
#: Add operation, value: "+"
add = "+"
```